### DISCUSSION TITLE
Επιλογή WM

### DISCUSSION THREAD

**Discussion by Axileaszervos** [courses-ionio/iv#19] [Q&A] [answered]
Καλησπέρα @epidrome, όσον αναφορά το δεύτερο παραδοτέο και τα WM, στις οδηγίες αναφέρετε τον i3 αλλά και τα [SOWN](https://github.com/dylanaraps/sowm#sowm-simple-shitty-opinionated-window-manager),[DWM](https://dwm.suckless.org/), ο καθένας μπορεί να χρησιμοποιήσει όποιον θέλει ή πρέπει να χρησιμοποιήσει κάποιο από τα `SOWN`,`DWM`? Μιας και στα assignments αναγράφονται ως παραδείγματα μόνο τα προαναφερόμενα WM(`SOWN`,`DWM`).

**Comment by epidrome** [comment, marked as answer]
Αυτά που αναφέρονται είναι ενδεικτικά, μπορείτε να χρησιμοποιήσετε όποιο θέλετε, ακόμη και πέρα από αυτά.
